docs/snippets/svelte/msw-addon-configure-handlers-graphql.ts-4-9.mdx

```diff
@@ -10,6 +10,9 @@ import MockApolloWrapperClient from './MockApolloWrapperClient.svelte';
 
 const meta = {
   component: DocumentScreen,
+  render: () => ({
```

Comment:
same comment as in the other snippet regarding this potentially not being needed